static/bidder-info/driftpixel.yaml

```diff
@@ -0,0 +1,19 @@
+endpoint: "http://rtb.driftpixel.live?pid={{.SourceId}}&host={{.Host}}&pbs=1"
+maintainer:
+  email: "developer@driftpixel.ai"
```

Comment:
@driftpixelai We have sent an email to `developer@driftpixel.ai` for verification. Please reply to the mail with 'Received' message to complete the verification process.

Thank you!

Reply:
@driftpixelai  Please respond to this mail to complete the verification process.

Reply:
Hi @Sonali-More-Xandr, have you received the response?